lib/services/doctor-service.ts

```diff
@@ -48,6 +50,14 @@ class DoctorService implements IDoctorService {
 			this.$logger.error("Cannot get the latest versions information from npm. Please try again later.");
 		}
 
+		try {
```

Comment:
move this try-catch and the `if` statement inside `checkForDeprecatedShortImportsInAppDir` method